### PR TITLE
Temporarily remove Export Domains section

### LINKED ISSUE #860: Temporarily remove the Export Domains section
### Story

As a registrant
I want to go to the home page and not see the export domains button and section at the bottom of the page
so that I am not confused by the fact that this doesn't work for MVP

*Export domains section being referenced above:
image

### Acceptance Criteria

-Comment out the export domains section on home.html
-comment out any tests that are affected by this if applicable
-add a TODO in the comment to add back in when this is implemented post MVP

### Additional Context

See this [slack thread](https://cisa-corp.slack.com/archives/C05BDEA3C11/p1691455212627079) for context

Note that this is being commented out just for MVP, as it is not a high priority.

### Issue Links

Other updates to Domain management dashboard in ticket https://github.com/cisagov/getgov/issues/861

### PR DESCRIPTION
**Pull request by therealslimhsiehdy**
#  #

## 🗣 Description ##
Temporarily remove the Export Domains section until post-MVP and add a TODO about it

**Before:**
Screenshot 2023-08-14 at 6 50 31 PM

**After:**
Screenshot 2023-08-14 at 6 50 55 PM

## 💭 Motivation and context ##
Making sure I know how to run the test suite:

To run all current 200+ tests: `docker-compose exec app ./manage.py test`
To run a specific test: `docker-compose exec app ./manage.py test registrar.tests.{test_file_name}.{class name}.{test name}`

Side note, I did run into errors when running the test suite such as: 
```
botocore.exceptions.NoCredentialsError: Unable to locate credentials

Traceback (most recent call last):
  File "/app/registrar/models/domain_application.py", line 494, in _send_status_update_email
    send_templated_email(
  File "/app/registrar/utility/email.py", line 55, in send_templated_email
    raise EmailSendingError("Could not send SES email.") from exc
``` 
But when I spoke to Zander about it he also sees the same thing and it's bc of keys so will just ignore for now!

And to close #860!

### CODE REVIEW

File: src/zap.conf
Comment (zandercymatics) [inline src/zap.conf:34]:
```suggestion
# Ignores suspicious word "TODO"
```
(Non-blocking) I'd recommend adding some additional context here (like above) as to what this addresses, though this is not an important change

**Review by zandercymatics — APPROVED** [review]
LGTM! Works perfectly both locally and on your sandbox

**Review by gabydisarli — APPROVED** [review]
lgtm!

**Comment by abroddrick** [comment]
@therealslimhsiehdy just fyi it looks like there were suggestions on this one. Just in case you missed, it people should approve even if there are small changes, so we have to check if there are any outstanding suggestions/or minor comments to address. Sorry if that was mansplaining, I know I have merged before seeing a suggestion/comment so just wanted to make sure you were away sometimes you have to scroll up. (in this case it's so minor it really doesn't matter)

**Comment by therealslimhsiehdy** [comment]
@abroddrick So sorry totally on me, and not mansplaining at all! I completely did not Zander's comment 🤦🏻‍♀️

**Pull request merged**
